### PR TITLE
Added runit init and log script

### PR DESCRIPTION
**Pull request by wratc**
Runs as user loki, needs lokinet-bootstrap run as loki before and logs to /var/log/lokinet.

### CODE REVIEW

File: contrib/runit/run [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec chpst -u loki:loki /usr/local/bin/lokinet ~loki/.lokinet/lokinet.ini 2>&1
```
Comment (majestrate) [inline contrib/runit/run:2]: shouldn't it be `~/.lokinet/lokinet.ini` instead of `~loki/.lokinet/lokinet.ini` or is this some runit-ism?

**Comment by majestrate** [comment]
if you can please also provide a readme on how to use this. thnx.

File: contrib/runit/run (hunk 1 above)
Comment (wratc) [reply contrib/runit/run:2]: It's a chpst-ism because home dir is still /root I think.

Comment (majestrate) [reply contrib/runit/run:2]: i see

**Comment by wratc** [comment]
Do I open a different mr for that? And should I put the two run files in it again?

**Comment by majestrate** [comment]
you can add it in a new commit to your branch

**Comment by majestrate** [comment]
do we have any updates on documentation on how to use this?

**Pull request closed without merging**